README.md

```diff
@@ -6,11 +6,55 @@
 
 
 # How to deploy to production (Kave Version):
+This repository holds the helm chart's custom values we used to deploy self-managed Kutt on Kubernetes.
 
+Also, we are using Terraform to provision a CloudSQL instance for the Kutt database.
 
+## Prerequisites
+### service-account.json
+At 1Password look for "Terraform Service Account for Kave Tech Services", copy and paste it at the `terrafom/kutt/` folder. Name it `service-account.json`.
 
+Once it is done, `cd` to the `terrafom/kutt/` folder and perform a `terraform init`.
 
+## Apply terraform
+```
+terraform apply
+```
+Wait until terraform finish.
+
+Go to GCP on secret manager, view the secret named url_shortener_password and copy the database password to vault kave-tech-services/show/url-shortener ->  DATABASE_PASSWORD key.
+
+Go to SQL instances and get the private ip of kutt-url-shortener instance and copy on helm/custom-values.yaml
+```
+externalPostgresql:
+  auth:
+    hostname: <Private IP here>
+```
+
+## Prepare the namespace to use helm 
+Kutt has been deployed in k8s Kave Tech Services' `services-prod` cluster using the official Helm Chart under the `url-shortener` namespace.
+
+If it's first time:
+See how to use ssl cert -> [Use Google-managed SSL Cert](https://loop.microsoft.com/p/eyJ3Ijp7InUiOiJodHRwczovL2p1bGlhZ3J1cC5zaGFyZXBvaW50LmNvbS8%2FbmF2PWN6MGxNa1ltWkQxaUlWaHdaemM1ZVVwSU1qQXRlVjlwVldkTVlqRkdkRUZEV0VOUWRXVllha1pLZG1zMlUwSlliMVJDY1ZweFJsYzFUR3RaVkhWUk5FRnpZMkoyVFhGM05Fa21aajB3TVRWTFdGaElObGhVVnpWTlVFcEhWbE5JVGtoTVVFMHlVbEJUV1ZGQ1JFZFpKbU05Sm1ac2RXbGtQVEUlM0QiLCJyIjpmYWxzZX0sInAiOnsidSI6Imh0dHBzOi8vanVsaWFncnVwLnNoYXJlcG9pbnQuY29tLzpmbDovci9jb250ZW50c3RvcmFnZS9DU1BfZjczYjk4NWUtNDcyMi00ZmRiLWIyZmUtMjUyMDJkYmQ0NWI0L2xhJTIwQmlibGlvdGVjYSUyMGRlJTIwZG9jdW1lbnRvcy9Mb29wQXBwRGF0YS9TaW4lMjB0JUMzJUFEdHVsbyUyMDE1Lmxvb3A%2FZD13OTUzYWNmM2QzYzJlNGVhOGIzNjM4NzY5NjUyY2JiZjkmY3NmPTEmd2ViPTEmbmF2PWN6MGxNa1pqYjI1MFpXNTBjM1J2Y21GblpTVXlSa05UVUY5bU56TmlPVGcxWlMwME56SXlMVFJtWkdJdFlqSm1aUzB5TlRJd01tUmlaRFExWWpRbVpEMWlJVmh3WnpjNWVVcElNakF0ZVY5cFZXZE1ZakZHZEVGRFdFTlFkV1ZZYWtaS2RtczJVMEpZYjFSQ2NWcHhSbGMxVEd0WlZIVlJORUZ6WTJKMlRYRjNORWttWmowd01UVkxXRmhJTmxJMVdqUTFTa3RNVWpSV1FraE1SMWswU0U1R1UxTmFUemRhSm1NOUpUSkdKbVpzZFdsa1BURW1ZVDFNYjI5d1FYQndKbkE5SlRRd1pteDFhV1I0SlRKR2JHOXZjQzF3WVdkbExXTnZiblJoYVc1bGNpWjRQU1UzUWlVeU1uY2xNaklsTTBFbE1qSlVNRkpVVlVoNGNXUlhlSEJaVjJSNVpGaEJkV015YUdoamJWWjNZakpzZFdSRE5XcGlNakU0V1dsR1dXTkhZek5QV0d4TFUwUkpkMHhZYkdaaFZsWnVWRWRKZUZKdVVrSlJNV2hFVlVoV2JGZEhjRWRUYmxweVRteE9RMWRIT1ZWUmJrWmhZMVZhV0U1VmVISlhWbEl4VlZSU1FtTXlUbWxrYXpGNFpIcFNTbVpFUVhoT1ZYUlpWMFZuTWxkR1VsaE9WVEZSVTJ0a1YxVXdhRTlUUlhoUlZGUktVMVZHVGxwVlZVcEZVakZySlRORUpUSXlKVEpESlRJeWFTVXlNaVV6UVNVeU1qbG1aR1JoWlRJM0xXRm1NbUV0TkdGaFl5MWhNVGxsTFRBelpURXhNbVJsT0RrMVlTVXlNaVUzUkElM0QlM0QiLCJyIjpmYWxzZX0sImkiOnsiaSI6IjlmZGRhZTI3LWFmMmEtNGFhYy1hMTllLTAzZTExMmRlODk1YSJ9fQ%3D%3D) 
+
+## Now let's use helm 
```

Comment:
If I run this command on the root of the project, the `.` means nothing. Where should I `cd` to?